src/EFTools/setup/EFToolsMsi/Defines.wxi

```diff
@@ -1,27 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Include>
-  <?define LCID = 1033?>
-  <?define Lang = enu?>
-
-  <?if $(var.VisualStudioVersion) = 11.0?>
-    <?define UpgradeCode = {27790FD4-0395-45E1-9309-447B91604CCA}?>
-    <?define FullProductName = !(loc.FullProductName_Dev11)?>
-    <?define ProductDescription = !(loc.ProductDescription_Dev11)?>
-    <?define ProductVersion = 11.2.$(var.BuildRevisionProductVersion)?>
-  <?elseif $(var.VisualStudioVersion) = 12.0?>
-    <?define UpgradeCode = {B9BF703A-2405-4A71-AFC5-38F59C11911D}?>
-    <?define FullProductName = !(loc.FullProductName_Dev12)?>
-    <?define ProductDescription = !(loc.ProductDescription_Dev12)?>
-    <?define ProductVersion = $(var.AssemblyFileVersion)?>
-  <?elseif $(var.VisualStudioVersion) = 14.0?>
```

Comment:
last question, are there any concerns with removing support for older Visual Studio versions?

Reply:
It's a good question.  At this point, we have accepted package versions that are higher than 14 (and indeed, are really in 17.7 or higher, given the contents of https://github.com/dotnet/ef6tools/blob/36aa191f56cbf07ca555ee1fdb07ed7eed1376be/Directory.Build.props).  If we need to go back and service earlier versions, we'll need to start from a previous commit.